addons/strongerFlashlights/CfgWeapons.hpp

```diff
@@ -0,0 +1,306 @@
+class cfgWeapons
+{
+    class InventoryFlashLightItem_Base_F;    
+    
+    class cup_acc_flashlight; //CUP Flashlight -surefire
+    
+    class ItemCore;
+    class ACE_acc_pointer_green;
+    class acc_flashlight_pistol;
+    
+/////////////////////////////////////////
+//////////Multimode Flashlight///////////
+/////////////////////////////////////////    
+    
+    class GVAR(CUP_LED_flashlight_Beam): cup_acc_flashlight
+    {
+        author="Carl, Chesheire";
+        scope=1;
+        displayName="Tactical LED Flashlight (Multi-Mode)";
+        descriptionUse="Mode: Focused White Beam. Ctrl+L to change Mode";
+        descriptionShort="Mode: Focused White Beam. Ctrl+L to change Mode";
+        MRT_SwitchItemNextClass="QGVAR(CUP_LED_flashlight)";
```

Comment:
Incorrect usage: `MRT_SwitchItemNextClass="QGVAR(CUP_LED_flashlight)";` -> `MRT_SwitchItemNextClass = QGVAR(CUP_LED_flashlight);` `QGVAR` adds the quotes